data/lua/core/map.lua

```diff
@@ -86,6 +115,22 @@ if wesnoth.kernel_type() ~= "Application Lua Kernel" then
 end
 
 if wesnoth.kernel_type() == "Game Lua Kernel" then
+	---Represents a reference to a single hex on the map
+	---@class terrain_hex
+	---@field x integer
+	---@field y integer
+	---@field fogged boolean
+	---@field shrouded boolean
+	---@field team_label? string|tstring
+	---@field global_label? string|tstring
+	---@field label? string|tstring
```

Comment:
`label` could do with documentation about what it does when there's are multiple labels on the hex.

Reply:
I documented this field, but do I need more on this point?